packages/machine-extractor/example-site/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.0",
   "scripts": {
     "dev": "next",
-    "build": "yarn add xstate-parser-demo@latest && yarn && next build",
+    "build": "yarn add @xstate/machine-extractor@latest && yarn && next build",
```

Comment:
q: why do you install a package here? 🤔

Reply:
Great question, this is from the example-site which can probably be dropped.